src/main/java/com/fabriccommunity/spookytime/mixin/AbstractSkeletonEntityMixin.java

```diff
@@ -1,9 +1,20 @@
 package com.fabriccommunity.spookytime.mixin;
 
 import com.fabriccommunity.spookytime.MixinHelpers;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.mob.AbstractSkeletonEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.world.LocalDifficulty;
+
+import com.fabriccommunity.spookytime.SpookyConfig;
+import com.fabriccommunity.spookytime.registry.SpookyItems;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Random;
 
 import net.minecraft.entity.EquipmentSlot;
 import net.minecraft.entity.mob.AbstractSkeletonEntity;
```

Comment:
Not a major issue, but these imports aren't ordered in any way. Consider grouping them like they are in the original code.